base-api/serverless.yml

```diff
@@ -0,0 +1,78 @@
+service: your-api-name
+provider:
+  name: aws
+  runtime: dotnetcore3.1
+  vpc:
+    subnetIds:
+      - subnet-00356e0650138f74f
```

Comment:
Are these subnet IDs the same for all the API environments? Wondering if these might change depending on the environment.

Reply:
As far as I'm aware they are

Reply:
👍

Reply:
@elcorbs Sorry, just seeing this. No, the IDs are different in our AWS accounts. Is there a way to set serverless to retrieve the IDs based on names, rather than hardcoding those?

Reply:
Ah ok, sorry! Yeah we can specify them by stage in the serverless.yml as well